coremltools/models/datatypes.py

```diff
@@ -112,19 +122,18 @@ def __init__(self, key_type = None):
 
         self.key_type = key_type
 
-        _DatatypeBase.__init__(self, "Dictionary",
-                "Dictionary(%s)" % repr(self.key_type), None)
+        _DatatypeBase.__init__(self, "Dictionary", "Dictionary({})".format(repr(self.key_type), None))
```

Comment:
There is a bug here, `None` should not be in `format`: should change to  `_DatatypeBase.__init__(self, "Dictionary", "Dictionary({})".format(repr(self.key_type)), None)`

Reply:
Can you please open a PR to fix it? Thanks.

Reply:
https://github.com/apple/coremltools/pull/582